internal/sync.go

```diff
@@ -458,7 +456,7 @@ func (s *syncGSuite) getGoogleGroupsAndUsers(googleGroups []*admin.Group) ([]*ad
 
 	for _, g := range googleGroups {
 
-		if s.ignoreGroup(g.Name) {
```

Comment:
Looks like a bug in the new release

Reply:
What does that mean?

Reply:
I mean this line in rc.9 probably should be `if s.ignoreGroup(g.Name) || !s.includeGroup(g.Name)` so I changed it.